src/plone/staticresources/configure-resources.zcml

```diff
@@ -0,0 +1,19 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:plone="http://namespaces.plone.org/plone">
+
+  <plone:static
+    directory="static/icons"
+    type="plone"
+    name="icons"
```

Comment:
Icons seems to be an empty directory. What is it for?

Reply:
that's the place where icons should go, that don't belong to an icon-set

Reply:
:+1: